db/keyval/etcd/etcd.conf

```diff
@@ -31,4 +31,11 @@ auto-compact: true
 
 # If ETCD server lost connection, the flag allows to automatically run the whole resync procedure
 # for all registered plugins if it reconnects
-resync-after-reconnect: false
\ No newline at end of file
+resync-after-reconnect: false
+
+# Allow to start without connected ETCD database. Plugin will try to connect and if successful, overall resync will
+# be called
+allow-delayed-start: true
+
+# Number of seconds between ETCD reconnect attempts. Has no use if `delayed start` is turned off
+reconnect-interval: 1
```

Comment:
This needs to be changed because our parses parses it like raw `time.Time` (`uint64`), meaning it's in nanoseconds.